test/vre/chapter31/schema/maximal-test.json

```diff
@@ -0,0 +1,75 @@
+{
+  "data": {
+    "daytimePhone": "8009098888",
+    "disabilities": "PTSD",
+    "disabilityRating": 10,
+    "email": "test@foo.com",
+    "employer": "work Inc",
+    "employerAddress": {
+      "street": "122 first st",
+      "street2": "apt 2",
+      "city": "Wichita",
+      "country": "USA",
+      "state": "KS",
+      "postalCode": "83443"
+    },
+    "eveningPhone": "8009098888",
+    "hospitalAddress": {
+      "street": "122 first st",
+      "street2": "apt 2",
+      "city": "Wichita",
+      "country": "USA",
+      "state": "KS",
+      "postalCode": "83443"
+    },
+    "hospitalName": "hname",
+    "jobDuties": "working",
+    "monthlyIncome": 400,
+    "newVeteranAddress": {
+      "street": "123 first st",
+      "street2": "apt 2",
+      "city": "Wichita",
+      "country": "USA",
+      "state": "KS",
+      "postalCode": "83443"
+    },
+    "previousPrograms": [{
+      "program": "education program",
+      "yearStarted": 1900,
+      "yearLeft": 1901
+    }],
+    "serviceFlags": {
+      "ww2": true
+    },
+    "serviceHistory": [{
+      "serviceBranch": "Army",
+      "dateRange": {
+        "from": "2001-01-01",
+        "to": "2002-01-01"
+      },
+      "dischargeType": "honorable"
+    }],
+    "vaRecordsOffice": "office",
+    "veteranAddress": {
+      "street": "122 first st",
+      "street2": "apt 2",
+      "city": "Wichita",
+      "country": "USA",
+      "state": "KS",
+      "postalCode": "83443"
+    },
+    "veteranFullName": {
+      "first": "first",
+      "middle": "middle",
+      "last": "last",
+      "suffix": "Sr."
+    },
+    "veteranDateOfBirth": "1970-03-03",
+    "veteranSocialSecurityNumber": "424242424",
+    "veteranVaFileNumber": "3423434",
+    "view:inHospital": "Y",
```

Comment:
I think these flags are booleans?

Reply:
Will update

Reply:
I see what the problem is- I was using `selectRadio` instead of `selectYesNo`